requirements/requirements-optionals.txt

```diff
@@ -1,9 +1,9 @@
 # Optional packages which may be used with REST framework.
 psycopg2-binary>=2.8.5, <2.9
 markdown==3.1.1
-pygments==2.4.2
-django-guardian==2.2.0
-django-filter>=2.2.0, <2.3
+pygments==2.7.1
+django-guardian==2.3.0
+django-filter>=2.4.0, <2.5
```

Comment:
Why are some requirements semantically ranged, and others aren't? Seems like it would make sense to do them all.